frb_example/pure_dart/rust/build.rs

```diff
@@ -18,9 +20,11 @@ fn main() {
         ..Default::default()
     };
     // get opts from raw opts
-    let opts = config_parse(raw_opts);
-    // run flutter_rust_bridge_codegen
-    for opt in &opts {
-        frb_codegen(opt).unwrap();
+    let configs = config_parse(raw_opts);
+
+    // generation of rust api for ffi
+    let all_symbols = get_symbols_if_no_duplicates(&configs).unwrap();
+    for (i, config) in configs.iter().enumerate() {
+        frb_codegen(config, &all_symbols, i + 1).unwrap();
```

Comment:
Can we remove the block_index argument here, and integrate it into the config and auto gen it in config_parse? Otherwise it is quite cumbersome to let end users to specify such index manually.

Reply:
Fixed in d4575dc056dad1167a62ed7e81f06703130370df.